backend/api/Services/InspectionService.cs

```diff
@@ -61,5 +66,36 @@ private IQueryable<Inspection> GetInspections()
         {
             return _context.Inspections.Include(inspection => inspection.InspectionFindings);
         }
+
+        public async Task<MissionRun?> GetMissionRunFromIsarStepId(string isarStepId)
+        {
+            return await _context.MissionRuns.Where(m => m.Tasks.SelectMany(t => t.Inspections)
+            .Any(i => i.IsarStepId == isarStepId))
+            .FirstOrDefaultAsync();
+        }
```

Comment:
I cannot see where we use this function?

Reply:
Removed, redundant now.